src/Microsoft.AspNetCore.Antiforgery/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -22,7 +21,8 @@ public class DefaultAntiforgeryTokenGenerator : IAntiforgeryTokenGenerator
             _additionalDataProvider = additionalDataProvider;
         }
 
-        public AntiforgeryToken GenerateCookieToken()
+        /// <inheritdoc />
+        public AntiforgeryToken GenerateCookieToken(HttpContext httpContext)
```

Comment:
Discussed more and agreed not to do this - we're not convinced that this change would do enough. We'll take a fresh look at this API if anyone asks for this kind of extensibility.